### DISCUSSION TITLE
Working with multiple listings across multiple pages

### DISCUSSION THREAD

**Discussion by WillemSleegers** [quarto-dev/quarto-cli#523] [Q&A] [answered]
Hi all,

I have a question about how to work with multiple listings. I'm working on converting my personal website from `blogdown` to `quarto`. On my [website](https://www.willemsleegers.com) I have a main page that contains multiple listings (a projects listing and a blog posts listing). I also have a dedicated page for each; I have a page to list all my projects and a page to list my blog posts.

While I have figured out how to create these listings, I did run into a problem. When I click on a category in one of my blog posts, I either go to the listing on my dedicated page or to the listing on my main page, depending on how I got to the blog post. If it links to the dedicated page, there is no issue, but if I go to the blog post from the main page and then click on one of the categories, it takes me back to the main page. I'd like it to always go to the dedicated page. It also interferes with the multiple listings I have on my main page.

I hope this is clear enough. If not, you can see the issue if you go here: https://venerable-creponne-1d27d6.netlify.app, click on the Understanding Regression (Part 1) post, followed by clicking on one of the categories in that post.

**Comment by dragonstyle** [comment, marked as answer]
Hi,

Thanks for writing this up so clearly- we're taking pains to remember where you came from and create functional links, but in this case we're actually working against you (and it makes sense). Let me give a little thought to how we might improve the linking behavior for a case like this (e.g. perhaps we should target the 'nearest' listing rather than the literal source listing). I'll post an update here...

**Reply by dragonstyle** [reply]
This should be resolved as of:
b133089f130d2f6371a951a95459579849948fcf

A build should be available later today!
